specification/metrics/listcost.md

```diff
@@ -0,0 +1,34 @@
+# List Cost
+
+The List Cost represents a charge inclusive of the impacts of free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). It is used for calculating savings based on rate optimization activities, by comparing it with [Effective Cost](#effectivecost). In this context, activities such as increased consumption leading to lower volume/tier-based rates, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models are not considered rate optimization activities.
+
+The ListCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal and MUST NOT contain null values. When a [ListUnitPrice](#listunitprice) is not null, multiplying the ListUnitPrice by [QuantityInPricingUnit](#quantityinpricingunit ) MUST produce the ListCost. In cases where the ListUnitPrice is null, the following applies:
```

Comment:
nit: Can we switch this to use bullets so we don't have a mix of paragraph + bullets? This would be the only place we have that, I believe.

Reply:
@flanakin @ijurica  could we move this down to supporting information: In this context, activities such as increased consumption leading to lower volume/tier-based rates, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models are not considered rate optimization activities.

Does BYOL-based need to be included in the definition?

Reply:
- Agree with the suggestion to relocate.
- List of excluded rate optimization based on the reduced rates and discounts mentioned in **Cloud FinOps - Rate optimization chapter** and the **BYOL** is one of them